uber/models.py

```diff
@@ -1912,6 +1810,9 @@ def initialize_db():
     This should be the ONLY spot (except for maintenance tools) in all of core ubersystem or any plugins
     that attempts to create tables by passing modify_tables=True to Session.initialize_db()
     """
```

Comment:
This block was moved to the ``initialize_db`` function so that it will pick up on models defined in plugins.